tools/package.json

```diff
@@ -19,7 +19,9 @@
   "author": "support@expo.dev",
   "license": "MIT",
   "dependencies": {
+    "@expo/cli": "^0.3.2",
```

Comment:
To be honest I'm not a big fan of installing local CLI packages as dependencies for expotools because they will have to be updated on each release — expotools should be independent. Can we avoid that?

Reply:
I think maybe the right thing to do would be to move the `npm.ts` module to `@expo/config-plugins`, so that expotools and expo CLI could both use it.  @EvanBacon might have an opinion here...

Reply:
`@expo/config-plugins` also applies to my previous comment. Imho expotools shouldn't depend on any package that exists in the workspace.

Reply:
It seems to me that because we must now use the local `npx expo` instead of the old global CLI, we have no choice but to depend on the local CLI package, and by extension config-plugins. Doing things this way gets us the benefit of a lot of fairly complex code without having to copy and paste it into expotools.

Reply:
After consulting with @brentvatne , who pointed out that `npx create-expo-app` should really be used for this, I reworked the whole flow so that the dependencies are no longer needed. Also, we can now use local Typescript template if we want, e.g.

```
et gba -n TestGenerateApp -o ~/code --useLocalTemplate --template expo-template-blank-typescript expo-updates expo-dev-launcher expo-dev-client expo-dev-menu expo-updates-interface
```

will give you a Typescript app but with the correct prebuild iOS and Android directories from the local bare-minimum template.